app/furniture/marketplace/marketplaces/edit.html.erb

```diff
@@ -1,7 +1,6 @@
 <%- breadcrumb :edit_marketplace, marketplace %>
 
 <%= render Marketplace::ManagementComponent.new(marketplace: marketplace) do %>
-  <%= render "form", marketplace: marketplace %>
```

Comment:
I think it's safe to just delete this form now, since the Marketplace has no attributes on it that can be set by the user any more.